source/appModules/shellexperiencehost.py

```diff
@@ -1,13 +1,47 @@
 #A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2015 NV Access Limited
+#Copyright (C) 2015-2019 NV Access Limited, Joseph Lee
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
 
+"""App module for Shell Experience Host, part of Windows 10.
+Shell Experience Host is home to a number of things, including Action Center and other shell features.
+"""
+
 import appModuleHandler
 from NVDAObjects.IAccessible import IAccessible, ContentGenericClient
+from NVDAObjects.UIA import UIA
+import controlTypes
+import ui
+
+class ActionCenterToggleButton(UIA):
+
+	# Somehow, item status property repeats when Action Center is opened more than once.
+	_itemStatusMessageCache = None
+
+	def _get_value(self):
+		return self.UIAElement.currentItemStatus
+
+	def event_UIA_itemStatus(self):
+		self.event_valueChange()
+
+	def event_valueChange(self):
+		# Do not repeat item status multiple times.
+		currentItemStatus = self.value
+		if currentItemStatus and currentItemStatus != self._itemStatusMessageCache:
+			ui.message(currentItemStatus)
+			self._itemStatusMessageCache = currentItemStatus

```

Comment:
I don't think this should be within the if block, otherwise if you toggle something to not pressed and then pressed, and the itemStatus only has 1 pressed value, then  when going back to pressed the itemStatusCache will match and it will say nothing.